Story1/task/make 30 default

Liliya-V opened the pull request.

Osama-Tahboub1 commented on app/js/checkAnswers.js:
```diff
@@ -64,18 +64,26 @@ document.querySelector('#finish').addEventListener('click', function(e) {
             let html = body.innerHTML
             html += '<p class="error_message text-danger">Please contact admin. Answers cannot be checked at present.</p>'
-            html += '<p class="error_message text-danger">' + result.message + '</p>'
+            // html += '<p class="error_message text-danger">' + result.message + '</p>'
```
This code is commented.

karimkefi replied:
updated

karimkefi commented on app/js/checkAnswers.js:
```diff
@@ -64,18 +64,26 @@ document.querySelector('#finish').addEventListener('click', function(e) {
             let answered = getAnswered(userAnswers, questionAmount)
             displayResult(result.score, percentResult, answered)
+
+            let body = document.querySelector('body')
```
made changes suggested

karimkefi commented on app/js/sendResults.js:
```diff
@@ -0,0 +1,52 @@
+
+
+// /**
```
removed function  which was just for testing

karimkefi commented on app/js/sendResults.js:
```diff
@@ -0,0 +1,52 @@
+// }
+
+
+function handleResponseFromAPI (response) {
```
added

Osama-Tahboub1 commented on app/css/resultPage.css:
```diff
@@ -35,30 +35,30 @@ body {
 
```
Why are you committing CSS here? you should only commit scss, this will need to be fixed.

karimkefi replied:
deleted the css file. working now

olyolyoly committed:
added time field to admin pannel

Liliya-V committed:
Validated a time input so its not zero ro null or empty,
Made 30 a default,
Made is to be sent to JSOn timed 60 so its seconds in a DB ( not minutes)

Liliya-V committed:
Merge branch 'Story1/task/time-input-field' into Story1/task/make-30-default

Liliya-V committed:
new user form validation
